sig-cloud-provider/2020-annual-report.md

```diff
@@ -0,0 +1,72 @@
+# 2020 - SIG Cloud Provider - Community Meeting Annual Reports
+---
+## Operational
+**How are you doing with operational tasks in sig-governance.md?**
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+Our README is accurate but we do not currently have a contributing.md file.
+
+**All subprojects correctly mapped and listed in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml)?**
+
+I believe so.
+
+**What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?**
+
+Our meetings tends to be small and quiet. We try to keep the meeting notes upto date during the meeting. 
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+_Same question as above but for working groups._
+
+We go ove the sub project updates during our primary biweekly meeting. 
+I believe the OWNERS files are all upto date.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+---
+## Membership
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+I believe so.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+?
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+?
```

Comment:
I feel like there are two answer to this:

- For changes to cloud provider infrastructure where all approvers can approve, I think things are going reasonably well.
- For changes to a specific cloud provider implementation, where an approval is required from someone outside the company, I think there is a bigger problem, particularly for VMWare and Google since our SIG Chairs (our most active approvers) are from those respective companies.  I feel like we need to work to work toward having more approvers so (e.g. get to the point where the defacto bus factor is at least 2 for any change). I don't think this needs to imply that the immediate goal should be to add more top level approvers-- getting more sub-tree approvers and expanding the ownership of existing sub-tree approvers seems like a good path forward.

Curious what others think.